src/test/shared/utilities/timeoutUtils.test.ts

```diff
@@ -99,29 +98,42 @@ describe('timeoutUtils', async function () {
         })
 
         it('Correctly reports elapsed time with refresh', async function () {
-            const longTimer = new timeoutUtils.Timeout(10)
-            clock.tick(5)
+            // TODO: fake timers `refresh` is not implemented correctly for the non-global case
```

Comment:
I found a bug with fake timers that I'll need to upstream a fix for. It only happens when using a different global object to fake.